src/Avalonia.Controls/Button.cs

```diff
@@ -227,6 +236,7 @@ protected override void OnPointerPressed(PointerPressedEventArgs e)
             {
                 PseudoClasses.Add(":pressed");
```

Comment:
Now we have this property, we probably shouldn't be adding the `:pressed` pseudoclass ourselves. There's a `PseudoClass` method that can be used to make a pseudoclass shadow a property: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/Control.cs#L111